core/src/apps/bitcoin/multisig.py

```diff
@@ -28,7 +29,8 @@ def multisig_fingerprint(multisig: MultisigRedeemScriptType) -> bytes:
         if len(d.public_key) != 33 or len(d.chain_code) != 32:
             raise DataError("Invalid multisig parameters")
 
-    pubnodes = sorted(pubnodes, key=lambda n: n.public_key + n.chain_code)
+    if multisig.pubkeys_order == MultisigPubkeysOrder.LEXICOGRAPHIC:
+        pubnodes = sorted(pubnodes, key=lambda n: n.public_key + n.chain_code)
```

Comment:
Add comment explaining why we sort the XPUBs.

Reply:
Done in https://github.com/trezor/trezor-firmware/pull/4351/commits/edd0fcab3d910c606952fee325a74fa3cfff424a and https://github.com/trezor/trezor-firmware/pull/4351/commits/c703cacdcce782dacc8b41c4302ea95ee0f0760a.

Reply:
Done in https://github.com/trezor/trezor-firmware/pull/4351/commits/4e52009e889fca6cc7f8f2905c9b3ea2250ea429.